flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/dml/RichSqlInsert.java

```diff
@@ -30,18 +30,22 @@
 import org.apache.calcite.sql.parser.SqlParserPos;
 
 import java.util.LinkedHashMap;
+import java.util.List;
 
 /** A {@link SqlInsert} that have some extension functions like partition, overwrite. **/
 public class RichSqlInsert extends SqlInsert implements ExtendedSqlNode {
 	private final SqlNodeList staticPartitions;
 
+	private final SqlNodeList keywords2;
```

Comment:
extendedKeywords will be better than meaningless number

Reply:
Name it `keywords2` just because there is already a `keywords` in super class

Reply:
Yeah i know, so i proposed to name it `extendedKeywords` which have a cleaner meaning.